.gitignore

```diff
@@ -41,3 +42,6 @@ docs/changes/
 docs/latex/
 libdrizzle-config
 test-suite.log
+
+# Keep empty directories:
+# Keep empty directories: >> .gitignore/.git*
```

Comment:
Not entirely sure, as it was add in the automatic conversion. Would suspect that it is related to the `git` not tracking empty dirs